cicd/metrics_test.csv

```diff
@@ -37,3 +37,8 @@ avg_over_time(testmetric0[24h]),now-90d,now,"131.45590577322233,129.262268230913
 max_over_time(testmetric0[48h]),now-90d,now,"131.45590577322233,131.45590577322233,159.53343696489338",approx
 changes(testmetric0[48h]),now-90d,now,"0,1,2",eq
 "predict_linear(testmetric0[48h], 100)",now-7d,now,"127.06355282392336,159.54968553618554",approx
+"(testmetric0) > 132",now-90d,now,"159.53343696489338",approx
+"(testmetric0) >= 131",now-90d,now,"131.45590577322224,159.53343696489338",approx
+"(testmetric0) < 131",now-90d,now,"127.06863068860355",approx
+"(testmetric0) <= 1000",now-90d,now,"131.45590577322224,127.06863068860355,159.53343696489338",approx
+"(testmetric0) != 1000",now-90d,now,"131.45590577322224,127.06863068860355,159.53343696489338",approx
```

Comment:
Can you also test `==`?

Also the promQL documentation you linked mentions these comparison operators can be followed by a `bool` option so the output will be `0` or `1` rather than acting as a filter. Is supporting that option planned for a separate PR?

Reply:
Yes, i think i will create a separate PR.

For the `==`, i think i need to modify the cicd test as well. `==` is to match a exact value. But due to precision issue of the floating point, its results will change everytime. So i may need to think a good way to test that.